### ISSUE TITLE
Confusing error message on --confirm init package download

### ISSUE THREAD

**Issue by Racer159** [defenseunicorns/zarf#923]
### Environment
Device and OS: System76 popOS
App version: v0.22.1
Kubernetes distro being used: k3d
Other: N/A

### Steps to reproduce
1. run `zarf init --confirm` without an init package

### Expected result
An error message that explains that I need to either run without confirm or provide an init file.

### Actual Result
An error message that says that it "Failed to download the init package"

### Visual Proof (screenshots, videos, text, etc)
![image](https://user-images.githubusercontent.com/3977569/196444780-602dcbb0-cdfc-4378-877d-07b4e59ecc07.png)

### Severity/Priority
Low

### Additional Context
This is mostly wording, but could be confusing for new users.

**Comment by riffingonsoftware** [comment]
This appears to have been resolved with commit 659742f2 wherein a fmt.Errorf was introduced. Later, in commit a3d8c1f1, this was moved to lang.ErrInitNotFound. This can currently be seen at initialize:109-110 brought about by the return at initialize:120.

**Comment by AustinAbro321** [comment]
Yeah the message is now
```
this command requires a zarf-init package, but one was not found on the local system. Re-run the last command again without '--confirm' to download the package
```
